mne/stats/regression.py

```diff
@@ -227,8 +228,13 @@ def linear_regression_raw(raw, events, event_id=None, tmin=-.1, tmax=1,
     solver : str | function
         Either a function which takes as its inputs the sparse predictor
         matrix X and the observation matrix Y, and returns the coefficient
-        matrix b; or a string. If str, must be ``'cholesky'``, in which case
-        the solver used is ``linalg.solve(dot(X.T, X), dot(X.T, y))``.
+        matrix b; or a string.
+        X is of shape (n_times, n_predictors * time_window_length).
+        y is of shape (n_channels, n_times).
+        If str, must be either ``'cholesky'``, in which
+        case the solver used is ``linalg.solve(dot(X.T, X), dot(X.T, y))``,
+        or ``'ridge'``, in which case importing `ridge_regression` from
+        sklearn is attempted; otherwise, defaults to the same solver as before.
```

Comment:
This is unclear to me. What do you mean by "defaults to the same solver as before." ?